src/site/twitch.tv/modules/chat/ChatList.vue

```diff
@@ -139,7 +140,7 @@ function onChatMessage(msg: ChatMessage, msgData: Twitch.AnyMessage, shouldRende
 		msg.setComponent(typeMap[0], { msgData: msgData });
 	}
 
-	if (msgData.type === MessageType.RESTRICTED_LOW_TRUST_USER_MESSAGE) {
+	if (msgData.type === MessageType.RESTRICTED_LOW_TRUST_USER_MESSAGE && showSuspiciousUser.value) {
```

Comment:
Low Trust User ≠ Suspicious User

In my opinion their should be another setting to enable or disable low trust user highlights.